lib/rest-client.js

```diff
@@ -8,7 +8,15 @@ var util         = require('util');
 
 var isWin        = process.platform === 'win32';
 var HOME         = process.env[isWin ? 'USERPROFILE' : 'HOME'];
-var defaultConf  = require(HOME + '/.bitpay/config.json');
+try {
+  var defaultConf = require(HOME + '/.bitpay/config.json');
+} catch (e) {
+  var defaultConf = {
+    "apiHost": "test.bitpay.com",
+    "apiPort": 443,
+    "forceSSL": true
+  }
```

Comment:
To avoid duplicating default values already present elsewhere, I suggest using this approach instead:

```suggestion
  var defaultConf = require('../config/test.json');
```

See: https://github.com/bitpay/node-bitpay-client/issues/92#issuecomment-513958789